src/elements/element.arc.ts

```diff
@@ -129,8 +126,10 @@ function pathArc(
   ctx.beginPath();
 
   if (circular) {
-    // The first arc segment from point 1 to point 2
-    ctx.arc(x, y, outerRadius, outerStartAdjustedAngle, outerEndAdjustedAngle);
+    // The first arc segments from point 1 to point a to point 2
+    const outerMidAdjustedAngle = (outerStartAdjustedAngle + outerEndAdjustedAngle) / 2;
```

Comment:
What's the reason for splitting this in two?

Reply:
Doing so prevents "shortcut" borders for slices with self-overlap. Without it, there are discrepancies like this:

**single-slice-circumference-405**
_with vs. without midpoint_
![single-slice-circumference-405-expected](https://user-images.githubusercontent.com/1199584/196467290-8a8700d8-13a1-43e7-a42a-27862b6aab03.png) ![single-slice-circumference-405-actual](https://user-images.githubusercontent.com/1199584/196467748-6d693dbd-c72b-4656-9897-85b7a81a412d.png)

**manual circumference=495**
_with vs. without midpoint_
![single-slice-circumference-495-midpoint](https://user-images.githubusercontent.com/1199584/196470989-544846f9-7992-4ea9-89c2-b1903a652878.png) ![single-slice-circumference-495-no-midpoint](https://user-images.githubusercontent.com/1199584/196471159-ba7c0b74-46ae-4acb-8326-4eaa0d07d9de.png)